src/mount.coffee

```diff
@@ -57,11 +56,18 @@ wrapOutport = (transactions, client, instance, port, queueName) ->
         fields:
           deliveryTag: group
       data: null
+
+    result =
+      id: group
```

Comment:
If `result` should have `.type`, then it should be initialized here, so this is clear